fluss-server/src/main/java/com/alibaba/fluss/server/tablet/TabletServer.java

```diff
@@ -400,6 +400,13 @@ private static void validateConfigs(Configuration conf) {
                             ConfigOptions.TABLET_SERVER_ID.key()));
         }
 
+        if (conf.get(ConfigOptions.BACKGROUND_THREADS) < 1) {
```

Comment:
`COORDINATOR_IO_POOL_SIZE` ?

Reply:
`TabletServer` use `BACKGROUND_THREADS ` as io param.  Here I just noticed that there was no such check before, so I added it